connectors/kafka/src/main/java/forklift/producers/KafkaForkliftProducer.java

```diff
@@ -0,0 +1,141 @@
+package forklift.producers;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import forklift.connectors.ForkliftMessage;
+import forklift.message.Header;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.specific.SpecificRecord;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.errors.SerializationException;
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Created by afrieze on 2/27/17.
+ */
+public class KafkaForkliftProducer implements ForkliftProducerI {
+
+    private final String topic;
+    private final KafkaProducer<?, ?> kafkaProducer;
+    static ObjectMapper mapper = new ObjectMapper().registerModule(new JavaTimeModule())
+                                                   .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    String stringSchema = "{\"type\":\"record\"," +
+                          "\"name\":\"ForkliftStringMessage\"," +
+                          "\"fields\":[{\"name\":\"forkliftMsg\",\"type\":\"string\"}]}";
+
+    String mapSchema = "{\"type\":\"record\"," +
+                       "\"name\":\"ForkliftMapMessage\"," +
+                       "\"fields\":[{\"name\":\"forkliftMapMsg\",\"type\":\"string\"}]}";
+
+    String jsonSchema = "{\"type\":\"record\"," +
+                       "\"name\":\"ForkliftJsonMessage\"," +
+                       "\"fields\":[{\"name\":\"forkliftJsonMsg\",\"type\":\"string\"}]}";
+
+    Schema parsedStringSchema = null;
+    Schema parsedMapSchema = null;
+    Schema parsedJsonSchema = null;
+
+    public KafkaForkliftProducer(String topic, KafkaProducer<?, ?> kafkaProducer) {
+        this.kafkaProducer = kafkaProducer;
+        this.topic = topic;
+        Schema.Parser parser = new Schema.Parser();
+        this.parsedMapSchema = parser.parse(mapSchema);
+        this.parsedStringSchema = parser.parse(stringSchema);
+        this.parsedJsonSchema = parser.parse(jsonSchema);
+    }
+
+    @Override
+    public String send(String message) throws ProducerException {
+        GenericRecord avroRecord = new GenericData.Record(parsedStringSchema);
+        avroRecord.put("forkliftMsg", message);
+        ProducerRecord record = new ProducerRecord<>(topic, null, avroRecord);
+        try {
+            kafkaProducer.send(record);
```

Comment:
We should probably have a callback on all of the `kafkaProducer.send` calls to at least _log_ errors returned from the broker.